include/CppUTest/StandardCLibrary.h

```diff
@@ -1,8 +1,10 @@
 
+/* Must include this first to ensure the StandardC include in CppUTestConfig still happens at the right moment */
+#include "CppUTestConfig.h"
```

Comment:
True, but it feels ugly to include CppUTestConfig when I actually want StandardC :)